config/default.yml

```diff
@@ -838,6 +838,11 @@ PrePush:
     command: ['ruby', '-Ilib:test', '-rminitest', "-e 'exit! Minitest.run'"]
     include: 'test/**/*_test.rb'
 
+  TestUnit:
+    enabled: false
+    description: 'Run test-unit'
+    command: ['rake', 'test']
```

Comment:
Thank you very much for your kind advise. You are right. I fixed `default.yml` which you taught.